### ISSUE TITLE
Add issues for settings page

### ISSUE THREAD

**Issue by MoSattler** [HandsOnLearning/the-project#185]
We need issues to replicate this page: https://app.habitify.me/settings/general

The goal is to create some issues that would enable to parallelize work, and have multiple people working on it at the same time. Make the issues as descriptive as possible and add links/screenshots. 
But dont dicatate too much on how to implement those features, that will be up to the assignee.

**Comment by limbovski** [comment]
I'm not sure what you mean here @MoSattler?

**Comment by SimonNord** [comment]
Do you mean like: "Create input fields for Settings-General"? @MoSattler

**Comment by MoSattler** [comment]
Sorry, just saw this. 

No, I mean basicly take a look at the habitify app, and see what we have to do to get there. 
Then breaking up the work in tasks, and creating issues here, as I have done in the past.

Does that make sense?

**Comment by SimonNord** [comment]
Yes, I understand, so the above example would be okey if there are more description and images

**Comment by MoSattler** [comment]
I think so. let's try and iterate if need be.
